iib/workers/config.py

```diff
@@ -89,6 +95,9 @@ class TestingConfig(DevelopmentConfig):
     iib_greenwave_url = 'some_url'
     iib_omps_url = 'some_url'
     iib_request_logs_dir = None
+    if os.getenv('IIB_ENABLE_CACHE', '').lower() == 'true':
+        iib_dogpile_backend = 'dogpile.cache.memcached'
```

Comment:
I don't like how I have to install and run memcached to run some of the unit tests. I think we can just assume that dogpile and the memcached library work and you can just mock out the interactions with dogpile for testing the caching decorator.

Reply:
@mprahl Yeah, dogpile is now mocked. Thanks for the suggestion.